server/mailer/reports/weekly-content.js

```diff
@@ -0,0 +1,135 @@
+const {
+  default: {
+    array: { partition },
+    date: { formatDateTime }
+  }
+} = require('@sandstreamdev/std');
+
+const templateStyles = require('../common/styles');
+const mailTemplate = require('./weekly-template');
+
+const styles = templateStyles({ maxWidth: 1024 });
+
+const tableRow = data => `<tr><td>${data}</td></tr>`;
+const tableHeaderCell = data => `<th style="${styles.thData}">${data}</th>`;
+const tableCell = data => `<td style="${styles.tdData}">${data}</td>`;
+
+const generateDataTable = ({ header, items, showAuthor, host }) => {
+  let content = '';
```

Comment:
consider adding a new line after let and const declarations across this PR. We are doing this in the whole project
Regarding to this rule:
https://eslint.org/docs/rules/newline-after-var